_pages/homepage.md

```diff
@@ -53,93 +53,144 @@ flow:
 
-            A walkthrough video of setting up the development environment is now available - [/resources/morello-platform-model-and-android-stack-walkthrough/](/resources/morello-platform-model-and-android-stack-walkthrough/).
+            A walkthrough video of setting up the development environment is available - [/resources/morello-platform-model-and-android-stack-walkthrough/](/resources/morello-platform-model-and-android-stack-walkthrough/).
 
             ### Morello Hardware Development Platform
 
-            The Morello hardware development platform will become available early in Q1 2022 and will be
-            supported by the same software stack. Note: Availability of hardware will be limited - platforms
-            will be restricted to partners involved in defined research activities.
+            The Morello hardware development platform becomes available from Q1 2022 onwards and will be supported by the same software stack. Availability of hardware will be limited - platforms will be restricted to partners involved in defined research activities.
+
+            Software releases supporting initial board availability may not include complete support. For example: Support for DDR & PCIe may not initially be performant. Satellite peripherals (such as audio and video IPs) and features such as power management may not be supported at inception. Support will evolve over time and specifics will be detailed via `release-notes.rst` in the Documentation repository.
+
+            Note that fimrware pre-installed on boards should be updated to the latest versions immediately after first validation of boot.
       - format: text
         style: bg-white p-4 my-3
         text_content:
           text: |-
-            ## CHERI LLVM Compiler with Morello Support
+            ## Morello software enablement
 
-            Morello is supported by LLVM-based open-source toolchains based on the CHERI Clang/LLVM toolchain
-            from the University of Cambridge. Please note, these are experimental
-            toolchains and as such features may be missing.
+            The diagram shows a high-level view of the software stacks targeting the Morello hardware and FVP platforms.
 
-            ### Android CHERI LLVM/Clang toolchain
+            ![](/assets/images/content/Morello_Software_Stacks_Diagram_ST2_V2.png){:.img-fluid}
 
-            Includes a C/C++ compiler (clang), linker (lld), debugger (lldb), various utilities (such as
-            assembler & disassembler) and run-time libraries.
+            These stacks and the supporting tooling are intended to provide a foundation for ecosystem research. Enabling collaboration on existing work packages and new work on alternate RTOS/OS environments, tools and workloads. Functionality will evolve in stages throughout the lifetime of the Morello Program.
```

Comment:
First full stop should be a comma? The second sentence has no verb.

Reply:
Ack. Let go with the comma. These stacks and the supporting tooling are intended to provide a foundation for ecosystem research, enabling...